community/contributing.md

```diff
@@ -25,47 +26,57 @@ To help you familiarise yourself with the source tree, we have listed some impor
+
+- Currently, the website does not provide documentation for individual releases, only for code in the unstable branch of the kvrocks repository, so please be careful not to document behaviour specific to older releases when contributing (if you need to do so, please mark the release separately to avoid confusion).
 - After making major changes to the documentation, we recommend that you preview the changed version locally first to avoid formatting errors; you can use `yarn start` to build and preview the page locally.
 - Do not copy from other copyrighted documents.
 
 ## Find tasks and start to contribute
 
 A major obstacle for people just starting out and wanting to get involved in the community can be the difficulty of knowing where to start.
+
 To solve this problem, here are some suggestions to help new contributors start with simple tasks to learn the structure of the code and participate in the community step by step:
+
 - Filter the [Issues](https://github.com/apache/incubator-kvrocks/issues) with `good first issue` or `help wanted` label, pick a task from there, and get involved. Note that `good first issue` indicates an easy task for newcomers, while `help wanted` has no fixed task difficulty.
 - Check out the Kvrocks roadmap in [Projects](https://github.com/apache/incubator-kvrocks/projects/2) and [Discussions](https://github.com/apache/incubator-kvrocks/discussions) and pick the parts you are familiar with to contribute.
 - Keep up to date with Redis developments and feel free to contribute any features that exist in Redis and are missing in Kvrocks.
 
 ## Participate in the community and become a committer
 
-As you find tasks to start participating in the community, you will gradually learn about the workings and practices of the community.
-Here are some tips to help you feel more comfortable in the community:
+As you find tasks to start participating in the community, you will gradually learn about the workings and practices of the community. Here are some tips to help you feel more comfortable in the community:
+
 - Try to express your intentions in as many words as possible, and do not be afraid of words.
 - Talk to other members to get support before starting a new idea.
 
-After being involved in the community for a while, some members may want to learn how to become a committer, who has write access to the project (this is limited to merging PRs that have been approved and passed the CI, as kvrocks protects the main branch), and the ability to review PRs from others.
+After being involved in the community for a while, some members may want to learn how to become a committer, who has write access to the project (this is limited to merging PRs that have been approved and passed the CI, as kvrocks protects the unstable branch), and the ability to review PRs from others.
```

Comment:
I think in this line the  `kvrocks` word also should be capitalized.

Reply:
```suggestion
After being involved in the community for a while, some members may want to learn how to become a committer, who has write access to the project (this is limited to merging PRs that have been approved and passed the CI), and the ability to review PRs from others.
```

Reply:
We can even simplify the sentence :D